librad/src/keys.rs

```diff
@@ -59,27 +49,27 @@ pub trait SignError: error::Error + Send + Sync + 'static {}
 impl<T: error::Error + Send + Sync + 'static> SignError for T {}
 
 /// A device-specific signing key
-#[derive(Clone, Eq, PartialEq)]
+#[derive(Copy, Clone)]
```

Comment:
Wondering (for a long time, actually) if we should make `PublicKey` `Copy`, too. Would probably be a clippy-fest.

Reply:
Hahaha, yes, since it's only 32 bytes it's fairly cheap to copy.